velox/expression/tests/ExprTest.cpp

```diff
@@ -3157,6 +3159,45 @@ TEST_F(ExprTest, mapKeysAndValues) {
   ASSERT_NO_THROW(exprSet->eval(rows, context, result));
 }
 
+TEST_F(ExprTest, maskErrorByNull) {
```

Comment:
This test doesn't seem to sufficiently cover the new logic. How are you thinking about test coverage?